include/arch/riscv32/common/linker.ld

```diff
@@ -62,16 +64,16 @@ SECTIONS
 		*(.iplt)
 	}
 
-    GROUP_START(ROM)
+    GROUP_START(ROMABLE_REGION)
     _image_rom_start = .;
 
     SECTION_PROLOGUE(_VECTOR_SECTION_NAME,,)
     {
 		. = ALIGN(4);
 		KEEP(*(.vectors.*))
-    } GROUP_LINK_IN(ROM)
+    } GROUP_LINK_IN(ROMABLE_REGION)
 
-    GROUP_END(ROM)
+    GROUP_END(ROMABLE_REGION)
 
     GROUP_START(ROMABLE_REGION)
```

Comment:
Does the ROM-able region group need to be ended and started again?

Reply:
Yeah, It's a redundant, so I will patch it.